README.md

```diff
@@ -458,6 +458,41 @@ The following attributes are optional:
 
 
 
+### 35-pilot.ini / [Pilot] ###
+
+These sections describe the types of glideinWMS pilots that your site is willing to accept.
```

Comment:
```suggestion
These sections describe the size and scale of GlideinWMS pilots that your site is willing to accept.
```

Or something to that effect. Also, GlideinWMS is the official capitalization according to https://glideinwms.fnal.gov/doc.prd/install.html

Reply:
thanks, I was looking for something more descriptive than 'types' but couldn't think of it.